packages/common/src/lib/common/password.ts

```diff
@@ -90,16 +78,15 @@ const password = pw => {
         complexity: (password, callback) => {
             let result = false;
             if (typeof password === 'string') {
-                result =
+                result = !!(
                     password.length >= 8 && // minimum length is 8
                     password.match(/\d/) && // contains at least one digit
                     password.match(/[a-z]/) && // contains at least one lower case letter
-                    password.match(/[A-Z]/); // contains at least one upper case letter
+                    password.match(/[A-Z]/)
+                ); // contains at least one upper case letter
             }
             typeof callback === 'function' && callback(result);
```

Comment:
This can be written more concise as `callback?.(result)`, assuming callback is never something that's not a function or nullable.